package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.7.0",
```

Comment:
But the time that this is discovered is when a developer is attempting to do something else, such as this case with why I am working with this repo, and should not have to be dealing with this first in order to address their real concern.

The upgrade of this dependency should be an intentional thing that is explicitly undertaken as a separate effort that isn't a discovered one that is also then just handled.  The work I'm doing actually doesn't even require this upgrade to be completed - I just realized that things were out of date and took it upon myself to address it.  So this problem could have persisted for longer and longer until which point that not just the developer customer has a problem but then consumer customer does.  Once that happens how many versions behind is the code?  It requires even more effort to perform the upgrade and catch the differences between versions.

I get that the results of what I'm describing somewhat overlap with your own approach but with mine staying current is something that is intentionally sought out whereas with yours its addressed when it's a problem.  But I view using a version of the library that has associated code changes that have not yet been made as a problem, even if the code continues to work.

I have reverted the change